frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -1,56 +1,143 @@
-import {
-  getVmStatus,
-  VM_STATUS_IMPORTING,
-  VM_STATUS_V2V_CONVERSION_IN_PROGRESS,
-} from 'kubevirt-web-ui-components';
-
+import * as _ from 'lodash';
 import { asAccessReview, Kebab, KebabOption } from '@console/internal/components/utils';
-import { K8sKind, PodKind } from '@console/internal/module/k8s';
-import { isVMRunning } from '../../selectors/vm';
-import { startStopVmModal } from '../modals/start-stop-vm-modal';
-import { VMKind } from '../../types/vm';
+import { K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getName, getNamespace } from '@console/shared';
+import { confirmModal } from '@console/internal/components/modals';
+import * as React from 'react';
```

Comment:
Nit: vendor imports should come before application (including `@console/pkg`) imports.

```ts
import * as _ from 'lodash';
import * as React from 'react';


import { asAccessReview, Kebab, KebabOption } from '@console/internal/components/utils';
// other imports
```

(Looks like we're currently not enforcing this via `packages/.eslintrc.js` but existing code in `public` is already using this convention.)